in_toto/keylib.go

```diff
@@ -152,3 +154,63 @@ func VerifySignature(key Key, sig Signature, data []byte) error {
 
 	return nil
 }
+
+/* Parse ed25519 from private json uses the passed json string to populate and
```

Comment:
```suggestion
/* Parse ed25519 from private json uses the passed json string to populate an
```

Reply:
Also, I think per convention go function docstrings start with the function name.